anvio/interactive.py

```diff
@@ -191,6 +191,17 @@ def __init__(self, args, external_clustering=None):
 
             progress.end()
 
+        ContigsSuperclass.__init__(self, self.args)
+        self.init_splits_taxonomy(self.taxonomic_level)
+
+        if self.contigs_db_path:
+            self.contigs_db_variant = dbops.ContigsDatabase(self.contigs_db_path).meta['db_variant']
```

Comment:
This looks all very good, but can you please change this and test it to use the new function in `utils` that is added to the main branch by #1628?

Reply:
(you will have to merge the main branch into your branch first, of course :))